test/suites/integration/deregister_targets_test.go

```diff
@@ -42,33 +42,33 @@ var _ = Describe("Deregister Targets", func() {
 			service,
 			deployment,
 		)
-		time.Sleep(3 * time.Minute) // Wait for creation of VPCLattice resources
-
-		// Verify VPC Lattice Service exists
-		vpcLatticeService = testFramework.GetVpcLatticeService(ctx, pathMatchHttpRoute)
-
-		Expect(*vpcLatticeService.DnsEntry).To(ContainSubstring(
-			latticestore.LatticeServiceName(pathMatchHttpRoute.Name, pathMatchHttpRoute.Namespace)))
+		Eventually(func(g Gomega) {
```

Comment:
you probably need to split one large eventually into smaller ones for each API call